### PR TITLE
Checklist for course leaders

### PR DESCRIPTION
**Pull request by sofer**
@minaorangina @nofootnotes @nikhilaravi @msmichellegar Please pull this, [read it](http://fac6.foundersandcoders.org/leaders/index.html), edit it to taste, and get back me with any questions for Week 3.

### CODE REVIEW

**Comment by sofer** [comment]
@minaorangina @nofootnotes @nikhilaravi @msmichellegar more notes added.

**Comment by sofer** [comment]
@minaorangina @nofootnotes @nikhilaravi @msmichellegar I have updated the checklist again. Please pull when you have a mo'. Thanks.

File: leaders/README.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
+## Contact time
+
+### Monday
+Workshop, as required    
+17:00 Introduce invited speaker    
+
+### Tuesday
+10:00 Introduce morning challenge    
+10:45 Review morning challenge    
+13:00 Research presentations #1
```
Comment (iteles) [inline leaders/README.md:42]: We might need to revisit these timings

File: leaders/README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+### Thursday
+13:00 Run mid-point project reviews     
+17:00 Office hours    
+
+### Friday
+10:00 Code review kick-off    
+11:30 Stop code reviews give people a chance to respond to issues    
+13:00 Announce Stop, Go, Continue    
+13:30 Stop, Go, Continue review    
+14:15 Introduce Upstairs project presentation
```
Comment (iteles) [inline leaders/README.md:62]: Does this happen every week?

File: leaders/README.md [hunk 3]
```diff
@@ -1 +1,69 @@
...
+13:00 Run mid-point project reviews     
+17:00 Office hours    
+
+### Friday
+10:00 Code review kick-off    
+11:30 Stop code reviews give people a chance to respond to issues    
+13:00 Announce Stop, Go, Continue    
+13:30 Stop, Go, Continue review    
+14:15 Introduce Upstairs project presentation    
+14:45 Introduce weekend assignments
```
Comment (iteles) [inline leaders/README.md:63]: Are we giving homework every week? I vividly remember it being agreed (with FAC5) before the summer that students felt they needed the weekends to catch up on the previous week...

**Comment by iteles** [comment]
@sofer Just a couple of open questions before merging :blush:

File: leaders/README.md (hunk 2 above)
Comment (sofer) [inline leaders/README.md:62]: @iteles that's my plan.

File: leaders/README.md (hunk 3 above)
Comment (sofer) [inline leaders/README.md:63]: @iteles I think yes, but it should be kept light, i.e. mostly tidying up issues and doing a bit of reading and watching.

**Pull request merged**